packages/ColorPicker/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@igloo-ui/color-picker",
+  "version": "0.0.0",
```

Comment:
should we start at 1.0.0?

Reply:
we don't edit this manually. It'll be updated with the version package PR after this is committed. Since it's a minor version, it'll end up being 0.1.0